collectors/python.d.plugin/postgres/postgres.chart.py

```diff
@@ -525,19 +535,21 @@
         ELSE pg_current_xlog_location()
       END,
     replay_location) AS replay_delta
-FROM pg_stat_replication
+FROM pg_stat_replication psr
+LEFT OUTER JOIN pg_replication_slots prs on psr.pid = prs.active_pid
 WHERE application_name IS NOT NULL;
 """,
 }
 
 QUERY_STANDBY_LAG = {
     DEFAULT: """
 SELECT
-    application_name,
+    coalesce(prs.slot_name, psr.application_name) application_name,
     COALESCE(EXTRACT(EPOCH FROM write_lag)::bigint, 0) AS write_lag,
```

Comment:
```suggestion
    COALESCE(prs.slot_name, psr.application_name) application_name,
    COALESCE(EXTRACT(EPOCH FROM write_lag)::bigint, 0) AS write_lag,
```

How about keeping it uppercase?